Exception importing EQ grid

### Problem

Using this grid.xml:  https://t.co/Lgmqcx9DRm


We get this error when using the grid.xml importer in InaSAFE desktop

```
An error has occurred while executing Python code: 

InvalidLayerError 
Traceback (most recent call last):
  File "/Users/timlinux/.qgis2/python/plugins/inasafe/safe/gui/tools/shake_grid/shakemap_converter_dialog.py", line 184, in accept
    algorithm_filename_flag=True)
  File "/Users/timlinux/.qgis2/python/plugins/inasafe/safe/gui/tools/shake_grid/shake_grid.py", line 932, in convert_mmi_data
    return converter.mmi_to_raster(force_flag=True, algorithm=algorithm)
  File "/Users/timlinux/.qgis2/python/plugins/inasafe/safe/gui/tools/shake_grid/shake_grid.py", line 566, in mmi_to_raster
    self.create_keyword_file(algorithm)
  File "/Users/timlinux/.qgis2/python/plugins/inasafe/safe/gui/tools/shake_grid/shake_grid.py", line 878, in create_keyword_file
    raise InvalidLayerError()
InvalidLayerError


Python version: 2.7.10 (default, Oct 23 2015, 18:05:06) [GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] 
QGIS version: 2.13.0-Master Master, ed32f49 

Python Path:
/Applications/QGIS_2.13-dev.app/Contents/MacOS/../Resources/python/plugins/processing
/Applications/QGIS_2.13-dev.app/Contents/MacOS/../Resources/python
/Users/timlinux/.qgis2/python
/Users/timlinux/.qgis2/python/plugins
/Applications/QGIS_2.13-dev.app/Contents/MacOS/../Resources/python/plugins
/Library/Frameworks/SQLite3.framework/Versions/C/Python/2.7
/Library/Frameworks/GEOS.framework/Versions/3/Python/2.7/site-packages
/Library/Python/2.7/site-packages/numpy-override
/Library/Python/2.7/site-packages/matplotlib-override
/Library/Frameworks/GDAL.framework/Versions/1.11/Python/2.7/site-packages
/Library/Python/2.7/site-packages/pip-6.1.1-py2.7.egg
/Library/Python/2.7/site-packages/pycharm-debug.egg
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python27.zip
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-darwin
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac/lib-scriptpackages
/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-tk
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-old
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload
/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/PyObjC
/Library/Python/2.7/site-packages
/Users/timlinux/.qgis2//python
/Users/timlinux/.qgis2/python/plugins/inasafe
/Users/timlinux/.qgis2/python/plugins/inasafe/safe_extras/parameters
/Users/timlinux/.qgis2/python/plugins
/Users/timlinux/.qgis2/python/plugins/inasafe/safe_extras
C:/eclipse/plugins/org.python.pydev_3.6.0.201406232321/pysrc
/Library/Python/2.7/site-packages/IPython/extensions
/Users/timlinux/.qgis2/python/plugins/SGDiagramDownloader/third_party
/Users/timlinux/.qgis2/python/plugins/StreamFeatureExtractor/third_party
.
/Applications/QGIS_2.13-dev.app/Contents/Resources/python/plugins/fTools/tools
```


### Expected Behaviour

No crash on import

### Environment

### Proposed Solution

hi @timlinux 
So, I just found this.
The travis build said everything is passing. In fact, only not the tests that are marked as not slow which run by the travis.
And the `test_convert_grid_to_raster` that test the functionality of this error has failed. I ran it manually from my local pycharm. In fact, there are 4 failed test in the file.
So, we need to include all not slow test to the travis. But it will make the travis exceeding it time limit.

I have done a small check, and it seems it's not related with the keywords. It doesn't generate the tiff file. To check, I comment out this line:
```
        if not hazard_layer.isValid():
            raise InvalidLayerError()
```
in https://github.com/inasafe/inasafe/blob/develop/safe/gui/tools/shake_grid/shake_grid.py#L877. And it failed, because couldn't find the generated file.
```
AssertionError: File result : /var/folders/6w/gz0ptx316pj4_x38p2fsl9lw0000gn/T/inasafe/2016-03-17/ismailsunni/test/result_grid4NwC6Q-nearest.tif does not exist
```

The last time I worked with metadata/keywords, I checked all package test (including the slow one), and all are fine. After that, I didn't check for all slow test.

--- a/safe/gui/tools/shake_grid/shake_grid.py
+++ b/safe/gui/tools/shake_grid/shake_grid.py
@@ -412,30 +412,6 @@ class ShakeGrid(object):
 
         return vrt_path
 
-    def _add_executable_prefix(self, command):
-        """Add the executable prefix for gdal binaries.
-
-        This is primarily needed for OSX where gdal tools are tucked away in
-        the Library path.
-
-        :param command: A string containing the command to
-        which the prefix will be prepended
-        :type command: str
-
-        :return: A copy of the command with the prefix added.
-        :rtype: str
-
-        :raises: None
-        """
-
-        executable_prefix = ''
-        if sys.platform == 'darwin':  # Mac OS X
-            # .. todo:: FIXME - softcode gdal version in this path
-            executable_prefix = ('/Library/Frameworks/GDAL.framework/'
-                                 'Versions/1.9/Programs/')
-        command = executable_prefix + command
-        return command
-
     def _run_command(self, command):
         """Run a command and raise any error as needed.
 
@@ -446,7 +422,6 @@ class ShakeGrid(object):
 
         :raises: Any exceptions will be propagated.
         """
-        command = self._add_executable_prefix(command)
         try:
             my_result = call(command, shell=True)
             del my_result

